app/src/main/java/com/example/bigowlapp/viewModel/SignUpViewModel.java

```diff
@@ -19,26 +19,17 @@ public SignUpViewModel() {
     }
 
     // TODO: Create a default group upon user creation
```

Comment:
oops, you can remove this TODO. Left this by accident, its completed

Reply:
I removed it.